django/contrib/gis/gdal/raster/band.py

```diff
@@ -47,19 +56,80 @@ def pixel_count(self):
         """
         return self.width * self.height
 
+    _stats_refresh = False
+
+    def statistics(self, refresh=False, approximate=False):
+        """
+        Compute statistics on the pixel values of this band.
+
+        The return value is a tuple with the following structure:
+        (minimum, maximum, mean, standard deviation).
+
+        If approximate=True, the statistics may be computed based overviews, or
```

Comment:
Indeed, the word "on" was missing here. Rasters files can have "overviews" (basically pyramids of lower resolution version of the same raster) and/or internal tiling, which gdal can leverage to compute approximate statistics faster.